lms/static/js/spec/commerce/receipt_view_spec.js

```diff
@@ -19,17 +20,23 @@ define([
                 AjaxHelpers.respondWithJson(requests, responseData);
             };
 
-            mockRender = function(useEcommerceOrderNumber, isVerified) {
-                var requests, view, orderUrlFormat;
+            mockRender = function(useEcommerceOrderNumber, isVerified, requestInThemedSite) {
+                var requests, view, orderUrlFormat,
+                    actualRequestInThemedSite = requestInThemedSite;
+                if (typeof actualRequestInThemedSite === 'undefined') {
+                    actualRequestInThemedSite = 'False';
```

Comment:
@clintonb Most likely it would work with `actualRequestInThemedSite` (and corresponding parameters up the call chain) being a proper boolean, but this is closer to how LMS actually operates: it [renders a python boolean into a HTML tag](https://github.com/edx/edx-platform/blob/master/lms/templates/commerce/checkout_receipt.html#L55). When it happens, booleans are `str`ed, and their python string representation is "True" and "False` (i.e. titlecase).

I would suggest keeping it as now for two reasons:

* As I've said, it is closer to how LMS actually works
* We maintain a fork with this exact commit applied - it would be nice to not introduce the code drift.